Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -588,4 +595,12 @@ def verify_bintray_published(formulae_names)
       end
     end
   end
+
+  def check_bintray_mirror(name, url)
+    headers = curl_output("--connect-timeout", "15", "--head", url)[0]
+    status_code = headers.scan(%r{^HTTP\/.* (\d+)}).last[0]
+    return if status_code.start_with?("3")
+    opoo "The Bintray mirror #{url} is not reachable (HTTP status code #{status_code})."
+    opoo "Do you need to upload it with `brew mirror #{name}`?"
```

Comment:
That was the first thing I tried. It looked kinda hideous though, and the second line is easily ignored.

Nonstandard bump subject also sets the precedent for back-to-back opoos for the same issue:

```rb
          opoo "Nonstandard bump subject: #{orig_subject}"
          opoo "Subject should be: #{bump_subject}"
```

```
Warning: Nonstandard bump subject: imagemagick: bump homebrew mirror url for testing
Warning: Subject should be: imagemagick 7.0.5-404
```